yellowbrick/classifier/threshold.py

```diff
@@ -0,0 +1,351 @@
+            color_values = resolve_colors(
+                num_colors=3, colormap=self.colormap, color=self.color)
+        else:
+            color_values = get_color_cycle()
+
+        # unpack first three colors for visualization
+        recall_color, precision_color, queue_rate_color = color_values[:3]
+
+        if self.ax is None:
+            self.ax = plt.gca(xlim=[-1, 1], ylim=[-1, 1])
+
+        uniform_thresholds = np.linspace(0, 1, num=101)
+        uniform_precision_plots = []
+        uniform_recall_plots = []
+        uniform_queue_rate_plots = []
+
+        for data in self.plot_data:
+            uniform_precision = []
+            uniform_recall = []
+            uniform_queue_rate = []
+            for ut in uniform_thresholds:
+                index = bisect.bisect_left(data['thresholds'], ut)
+                uniform_precision.append(data['precision'][index])
+                uniform_recall.append(data['recall'][index])
+                uniform_queue_rate.append(data['queue_rate'][index])
+
+            uniform_precision_plots.append(uniform_precision)
+            uniform_recall_plots.append(uniform_recall)
+            uniform_queue_rate_plots.append(uniform_queue_rate)
+
+        lower_precision, median_precision, upper_precision = mstats.mquantiles(
+            uniform_precision_plots, prob=self.quantiles, axis=0)
+        lower_recall, median_recall, upper_recall = mstats.mquantiles(
+            uniform_recall_plots, prob=self.quantiles, axis=0)
+        lower_queue_rate, median_queue_rate, upper_queue_rate = mstats.mquantiles(
+            uniform_queue_rate_plots, prob=self.quantiles, axis=0)
+
+        self.ax.plot(
+            uniform_thresholds, median_precision, color=precision_color)
+        self.ax.plot(uniform_thresholds, median_recall, color=recall_color)
+        self.ax.plot(
+            uniform_thresholds, median_queue_rate, color=queue_rate_color)
+
+        self.ax.fill_between(
+            uniform_thresholds,
+            upper_precision,
+            lower_precision,
+            alpha=0.5,
+            linewidth=0,
+            color=precision_color)
+        self.ax.fill_between(
+            uniform_thresholds,
+            upper_recall,
+            lower_recall,
+            alpha=0.5,
+            linewidth=0,
+            color=recall_color)
+        self.ax.fill_between(
+            uniform_thresholds,
+            upper_queue_rate,
+            lower_queue_rate,
+            alpha=0.5,
+            linewidth=0,
+            color=queue_rate_color)
+
+        self.ax.axis('auto')
+
+        return self.ax
+
+    def finalize(self, **kwargs):
+        """Finalize executes any subclass-specific axes finalization steps.
+        The user calls poof and poof calls finalize.
+
+        Parameters
+        ----------
+        kwargs: generic keyword arguments.
+
+        """
+        # Set the title
+        if self.title is not None:
+            self.set_title(self.title)
```

Comment:
Title handled by super class if you want to call super on final. Also recommend you give a generic title in case one doesn't exist like "Threshold Plot of Binary Classifier" or something like that.